src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('table');
```

Comment:
Ensure that the table element exists in the DOM before querying it. If the table is not present, `querySelector` will return `null`, which could lead to errors when attempting to access `querySelectorAll` on `null`.